src/App.tsx

```diff
@@ -1,14 +1,39 @@
-/* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions as todosActions } from './features/todos';
+
+export const App = () => {
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const loadTodos = async () => {
+    setIsLoading(true);
+    try {
+      const loadedTodos = await getTodos();
+
+      dispatch(todosActions.setTodos(loadedTodos));
+    } catch {
+      throw new Error('Loading todos error');
```

Comment:
FYI: such thrown and unhandled errors will cause the page to crash. in such cases, it's better to show an error message somewhere, etc.